spec/integration/plos_details_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper.rb'
+
+describe PlosDetails, :vcr do
+  let(:plos_paper) do
+    create(:plos_paper, id: 1, reader: 0, doi: '10.1371/journal.pone.0089204')
+  end
+  let(:job) {PlosDetails.new}
+
+  it 'updates the view count' do
+    before = plos_paper.reader
+
+    VCR.use_cassette 'plos_detail' do
+      job.perform plos_paper.id
+    end
```

Comment:
If I remove these lines I get:

           There is currently no cassette in use. There are a few ways [...]

doesn't matter whether the cassette is named plos_detail.yml, plos_details_spec.yml etc. Missing some config somewhere?